### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jcrivenaes**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -10,8 +10,7 @@ For FMU users
 **A normal user will not need to install anything!**
 
 The stable and bleeding release of the scripts is distributed
-within Equinor, either through
-/project/res or Komodo. The front-end to run is the ``fmuconfig`` script:
+within Equinor, either through Komodo. The front-end to run is the ``fmuconfig`` script:
```
Comment (tralsos) [inline docs/installation.rst:13]: You could delete "either" ?

File: docs/usage.rst [hunk 2]
```diff
@@ -2,77 +2,106 @@
...
+   # run command for creating IPL version if needed (+ ert tmpl version; ipl.tmpl)
+   fmuconfig $MASTER --rootname $ROOTNAME --mode ipl --destination $OUTFOLDER \
+         --template $OUTFOLDER --tool rms
 
 
 Run from python inside or outside RMS
 -------------------------------------
 
-The config can also be ran from a python script, e.g. inside RMS. In that case you
+In RMS there two options to run global variables:
```
Comment (tralsos) [inline docs/usage.rst:61]: there are

**Review by tralsos — APPROVED** [review]
Looks good, thanks.

File: docs/usage.rst [hunk 3]
```diff
@@ -2,77 +2,106 @@
...
+* Run shell script above using system command (recommended)
+* Run from RMS python
+
+For shell script for RMS, use a system command job as this:
+
+.. code-block:: shell
+
+   cd ../../fmuconfig/bin; run_external sh global_variables_update.sh
+
+The config can also be ran from a python script inside RMS. In that case you
```
Comment (berland) [inline docs/usage.rst:72]: ran -> run

File: docs/usage.rst [hunk 4]
```diff
@@ -2,77 +2,106 @@
...
+The ``bin`` has shell scripts that process the input to a set of YAML and/or IPL
+which is stored at ``output``. The files in output shall never be edited by hand.
+
+
 Run from script
 ---------------
 
-The fmu.config module is accessed through a script, which is run like this::
+The ``fmu.config`` module is accessed through a script, and assuming the file structure
+above, to be run like this:
```
Comment (berland) [inline docs/usage.rst:28]: -> "above, is to be run like this:"

File: docs/usage.rst [hunk 5]
```diff
@@ -2,77 +2,106 @@
...
-   root_ipl = 'global_variables'
 
-   root_yml = 'global_variables_rms'  # to avoid name conflict
+   MASTER = "../../fmuconfig/input/global_master_config.yml"
+   OUTPUT = "../../fmuconfig/output"
+   ROOT = "global_variables"
 
-   cfg.parse(global_master)
+   def cleanup():
+      for fil in Path(OUTPUT).glob(ROOT + "*"):
```
Comment (berland) [inline docs/usage.rst:87]: fil  -> file ?

**Pull request merged**